### PR TITLE
Remove trophies from the 20-hour course

### PR DESCRIPTION
**Pull request by joshlory**
**Important: Don't merge before classes are finished for the year.**

This change removes all traces of Concept Trophies from the codebase.  I've left the database tables intact in case we need to revert for some reason.  If all goes well they'll be removed in a future migration.

concept-trophies

The goal is to begin reducing the amount of code we need to support for a limited number of users.  Trophies were only ever used in the legacy 20-hour course and don't do a great job of tracking student learning.

Next I plan to remove the `Concept` model (it's been replaced by `ConceptDifficulties`).

### CODE REVIEW

**Comment by ashercodeorg** [comment]
At a high level, LGTM, though I haven't looked at the PR carefully (nor will I until we get the go ahead from PM). As noted offline, we might consider doing this PR in two steps: removing trophies from the UI (though keep tracking them) and removing the tracking of trophies.

File: apps/src/feedback.js [hunk 1]
```diff
@@ -626,11 +611,6 @@ FeedbackUtils.prototype.getFeedbackMessage_ = function (options) {
...
             message = message + reinfFeedbackMsg;
           }
-        } else if (options.numTrophies > 0) {
-          message = finalLevel ? msg.finalStageTrophies(msgParams) :
-                                 stageCompleted ?
-                                    msg.nextStageTrophies(msgParams) :
-                                    msg.nextLevelTrophies(msgParams);
         } else {
           var nextLevelMsg = (options.appStrings && options.appStrings.nextLevelMsg) ||
               msg.nextLevel(msgParams);
```
Comment (Hamms) [inline apps/src/feedback.js:603]: does this need to be updated to deal with the lack of a `numTrophies` param?

File: dashboard/app/views/reports/prizes.html.haml [hunk 2]
```diff
@@ -26,12 +27,12 @@
       %br/
 
   - if current_user.teacher_prize
-    %h4= t('redeem_prizes.msg.teacher.earned', num_trophies: Trophy::TROPHY_MAX)
+    %h4= t('redeem_prizes.msg.teacher.earned', num_trophies: trophy_max)
```
Comment (Hamms) [inline dashboard/app/views/reports/prizes.html.haml:30]: why not remove this line altogether?

**Comment by Hamms** [comment]
Given that this will remove all historical trophy data, do we want to back that up somewhere?

**Comment by Hamms** [comment]
This LGTM, although this seems like a good candidate for a `[test all] [test ui]` PR

File: dashboard/app/views/reports/prizes.html.haml (hunk 2 above)
Comment (joshlory) [inline dashboard/app/views/reports/prizes.html.haml:30]: I want to avoid changing this legacy prizes page — I expect this page will soon go away entirely.

**Comment by joshlory** [comment]
I've left the database tables intact in case we need to revert for some reason.  (However new trophies will no longer be tracked.)

**Pull request merged**